<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GSolveWidget</class>
 <widget class="QWidget" name="GSolveWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="2,0">
   <item row="0" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QCustomPlot" name="tabR">
      <attribute name="title">
       <string>Red</string>
      </attribute>
     </widget>
     <widget class="QCustomPlot" name="tabG">
      <attribute name="title">
       <string>Green</string>
      </attribute>
     </widget>
     <widget class="QCustomPlot" name="tabB">
      <attribute name="title">
       <string>Blue</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
